noxfile.py

```diff
@@ -128,6 +128,30 @@ def build_docs(session):
 
                 output.write(f"   {path}\n")
 
+                # TODO: Explain
+                with open(file) as fd:
+                    content = fd.read()
+
+                if "Download Conda environment file" in content:
+                    # do not add the download link twice
+                    pass
+                else:
+                    lines = content.split("\n")
+                    with open(file, "w") as fd:
+                        for line in lines:
+                            if "sphx-glr-download-jupyter" in line:
+                                # add the new download link before
+                                fd.write(
+                                    """
+    .. container:: sphx-glr-download
+
+      :download:`Download Conda environment file: environment.yml <environment.yml>`
+"""
+                                )
+
+                            fd.write(line)
+                            fd.write("\n")
```

Comment:
Seems to work very well. I don't know if like that this is here. I rather would like to put this as an extensions sphinx-gallery directly.

I will explore this.

Reply:
Did you find a way to do this? Or should we merge this version for now?

Reply:
I have an idea how to do this within the conf.py. But, we can merge this and I change this later!